buildbot/buildbot/server.py

```diff
@@ -46,9 +47,9 @@ def _datetime_filter(value, withtime=True):
         return ''
     dt = datetime.fromtimestamp(value)
 
-    fmt = '%B %d, %Y'
+    fmt = '%m/%d/%y'
     if withtime:
-        fmt += ', %I:%M %p'
+        fmt += ', %H:%M'
```

Comment:
This is definitely more legible than the old one, but if we're going for filename-like mental sortability, how about something like `%Y-%M-%d %H:%M`, i.e., approximately [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)?

Reply:
Fixed.